frontend/src/components/pages/search/hooks/useDateFilter.ts

```diff
@@ -0,0 +1,24 @@
+import { DateFilter } from 'modules/filters/interface';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+
+export const useDateFilter = (): {
+  dateFilter: DateFilter;
+  setDateFilter: (dFilter: DateFilter) => void;
+} => {
+  const initialOptions = useRouter().query;
+
+  const beginDate = initialOptions.beginDate ? initialOptions.beginDate.toString() : 'null';
```

Comment:
oh well i see, why don't you use `null` as null type? :D

Reply:
I try null type but not working as I've expected so I decided to use an empty string and it's ok :)